hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/calcite/parse/ParserOperationsTest.java

```diff
@@ -97,14 +106,6 @@ public void testUnsupportedWhereScalar() {
         );
     }
 
-    @Test
-    public void testUnsupportedOrderBy() {
```

Comment:
Perhaps we need tests for unsupported features (LIMIT/OFFSET, NULLS FIRST/LAST)

Reply:
Added in https://github.com/hazelcast/hazelcast/pull/18013/commits/5ac898c968033856201ea1e715b5a135aaea8198